src/index.css

```diff
@@ -1,4 +1,294 @@
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.editBtn {
+  padding-left: 1.063rem;
+  border: none;
+}
+
+.progressContainer {
+  display: flex;
+  justify-content: space-between;
+  width: 23.625rem;
+  align-items: center;
+}
+
+.circleProgress {
+  display: flex;
+  align-items: center;
+  padding-right: 30px;
+  border-right: 1px solid #e8e8e8;
+}
+
+.circle {
+  height: 4.25rem;
+}
+
+.circleProgressText {
+  display: flex;
+  flex-direction: column;
+}
+
+.circleProgressText > p {
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: 0.68rem;
+}
+
+.progressT1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progressT2 {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapterProgress {
+  display: flex;
+  flex-direction: column;
+}
+
+.chapterText {
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.chapterNo {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+  margin-top: 0.438rem;
+  margin-bottom: 1.438rem;
+}
+
+.updateBtn {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border-width: 0;
+}
+
+.addBook {
+  margin-top: 2.5rem;
+  border-top: 1px solid #e8e8e8;
+  padding-top: 1.813rem;
+}
+
+.addBook > h2 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-top: 0;
+  margin-bottom: 1.188rem;
+}
+
+.addBook > form {
+  display: flex;
+  justify-content: space-between;
+}
+
+.addBook > form > input {
+  padding: 0.813rem 15rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  margin-right: 2.125rem;
+}
+
+.addBook > form > button {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border-width: 0;
+}
```

Comment:
- [ ] Kindly increase the `padding` of the button and transform the text to `uppercase` for getting a replica of the Zeplin's design 🤓⛏

Your design | Given design
---| ---|
![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/333eeafe-76b2-400b-b317-e5d46cbb2f41) | ![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/e8694576-02d3-48d8-a3ae-a507082c9464)